docs/user-event/intro.mdx

```diff
@@ -11,36 +11,35 @@ events that would happen if the interaction took place in a browser.
 
 These docs describe `user-event@14`. We recommend updating your projects to this
 version, as it includes important bug fixes and new features. You can find the
-docs for `user-event@13.5.0` [here](../ecosystem-user-event.mdx).
+docs for `user-event@13.5.0` [here](../ecosystem-user-event.mdx), and the changelog for the release [here](https://github.com/testing-library/user-event/releases/tag/v14.0.0).
 
 :::
 
 While most examples with `user-event` are for `React`, the library can be used
 with any framework as long as there is a DOM.
 
-## Difference to `fireEvent`
+## Differences with `fireEvent`
 
 The built-in [`fireEvent`](dom-testing-library/api-events.mdx#fireevent) is a
-utility to easily dispatch events. It dispatches exactly the events you tell it
-to and just those - even if those exact events never had been dispatched in a
-real interaction in a browser.
+utility for easily dispatching events. It dispatches exactly, and only, the events you tell it
+to - even if those exact events would not have been dispatched in a
+real browser interaction.
 
-`user-event` on the other hand dispatches the events like they would happen if a
-user interacted with the document. That might lead to the same events you
-previously dispatched per `fireEvent` directly, but it also might catch bugs
+`user-event` on the other hand dispatches events as they would be dispatched by a user interacting with the document in a browser. That might lead to the same events you previously dispatched per `fireEvent` directly, but it also might catch bugs
 that make it impossible for a user to trigger said events. This is
 [why you should use `user-event`](https://ph-fritsche.github.io/blog/post/why-userevent)
 to test interaction with your components.
```

Comment:
_exactly, and only,_ doesn't greatly clarify what's going on here, in my opinion. Let's try reworking this paragraph completely, like this:

```suggestion
`fireEvent` simulates _browser events_, whereas `user-event` simulates full `interactions`, which may fire multiple events and do additional checks along the way.

Testing Library's built-in [`fireEvent`](dom-testing-library/api-events.mdx#fireevent) is a lightweight wrapper around the browser's low-level dispatchEvent API, which allows developers to trigger any event on any element. This is often sufficient for testing simple behavior, e.g., checking that a click event triggers the right callback, but there are cases where the browser does more than just trigger one event for one interaction, for example, when a user types into a text box, the element is first focused, and then input events are fired as they type. `user-event` also adds visibility and interactability checks before triggering the flow, to simulate how a user wouldn't click an element they can't see, and the browser wouldn't let them type in a disabled text box.
```

Reply:
@alexkrolick , I like it! I'll apply the suggestion (though manually, as attempting to just accept the suggested edit produces a "Applying suggestions on deleted lines is not currently supported" error). 

@ph-fritsche , this removes the line we were discussing entirely, which perhaps resolves our problem. Do you feel good about this solution?

Reply:
Rewriting the discussed sentence or the complete paragraph might just be solution.
But I feel this variant is a step back regarding the recommendation our users might read into this.

We've had some discussions about what we want to recommend for simulating any interactions before.
`userEvent` wasn't mature enough, so we pointed people towards `fireEvent` and mentioned that there is also `userEvent` doing more. v14 tackled all the issues being referenced every time this was discussed.
So when we talked about it again prior to releasing v14, the consensus was:
*People should use `userEvent` for testing any user input nowadays.
`fireEvent` works as an escape hatch for edge cases which are not implemented yet and for library authors who need control over the events and their properties to e.g. test their event system.*

> This is often sufficient for testing simple behavior, e.g., checking that a click event triggers the right callback

We want to communicate the opposite: Usually it is not a good idea to test simple behavior with `fireEvent` as it poses the risk to just repeat false assumptions in component and test.
The linked blog post contains an example for how testing an implementation detail (the exact event) provides a false-negative test although the presumably tested interaction is made impossible by a bug.
It also hints at how testing a perceived event (`change`) although working might create a test that is unrelated to the also working component - which might result in a very confusing scenario if one of them breaks one day.

Reply:
@ph-fritsche OK, how about this then. Removed that sentence:

https://github.com/testing-library/testing-library-docs/pull/1086/files#r897024155

Reply:
@alexkrolick I think that the idea to make this section less abstract might be the right choice.

I tried to make it even more explicit/"correct" in the description of user-event and re-added the link the post with the example, so that the section ends with an explicit recommendation and the option to read more about this if someone cares.

What do you think?

https://github.com/meatnordrink/testing-library-docs/compare/patch-1...testing-library:pr-1086/phf

Reply:
That version sounds good to me

Reply:
Do you want to accept that patch and merge this then?

Reply:
@meatnordrink I went ahead and merged the change into [meatnordrink:patch-1](https://github.com/meatnordrink/testing-library-docs/commits/patch-1)

Reply:
Thanks, @ph-fritsche  - sorry for not replying, I was on vacation (and camping) last week!